desktop/core/src/desktop/js/apps/admin/Configuration/ConfigurationTab.tsx

```diff
@@ -0,0 +1,157 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useState, useEffect, useMemo } from 'react';
+import { Spin, Alert } from 'antd';
+import { i18nReact } from '../../../utils/i18nReact';
+import './Configuration.scss';
+import { DEFAULT_SELECTED_APP, GET_CONFIG_API } from './utils';
+import AdminHeader from '../AdminHeader';
+import { ConfigurationValue } from './ConfigurationValue';
+import { ConfigurationKey } from './ConfigurationKey';
+
+interface App {
+  name: string;
+  has_ui: boolean;
+  display_name: string;
+}
+
+export interface ConfigValue {
```

Comment:
Can you find a more precise name for this? There could be many type of config values in Hue.

But also please verify that these interfaces (or types) are not already defined elsewhere. Check out 
`interface HueConfig` in `desktop/core/src/desktop/js/config/types.ts`

Reply:
Changed `ConfigValue` to `AdminConfigValue`.